jax_distribution/README.md

```diff
@@ -6,19 +6,15 @@ Inside of the `jax_distribution` folder, run:
 ### Python virtual env
 - venv:
     ```
-    python -m venv mava_30
-    source mava_30/bin/activate
+    make build_venv
     ```
 - OR conda:
     ```
     conda create -n mava_30 python=3.9
     conda activate mava_30
+    pip install --upgrade pip setuptools
+    pip install -r requirements.txt
```

Comment:
I think we need to mention that requirement is `jax_distribution/requirements.txt` since we may have another one somewhere else

Reply:
Great point! I updated it now.